### PR TITLE
Consistency in doc example

### PR DESCRIPTION
**Pull request by adam-meya**

### CODE REVIEW

**Comment by ClemDoum** [comment]
@adam-meya thanks for your contribution. We merge PR on `develop`, I changed the base of your PR but can you rebase your branch on `develop`?

**Comment by adam-meya** [comment]
Is the rebase necessary? I'm seeing the appropriate changes after you changed the PR base.

File: docs/source/tutorial.rst [hunk 1]
```diff
@@ -325,4 +325,4 @@ And load it:
 
 .. _sample dataset: https://github.com/snipsco/snips-nlu/blob/master/samples/sample_dataset.json
 .. _sample configurations: https://github.com/snipsco/snips-nlu/blob/master/samples/configs
-.. _english one: https://github.com/snipsco/snips-nlu/blob/master/samples/configs/config_en.json
\ No newline at end of file
+.. _english one: https://github.com/snipsco/snips-nlu/blob/master/samples/configs/config_en.json
```
Comment (adrienball) [inline docs/source/tutorial.rst]: Could you please discard this change? It's probably due to an automatic formatting

**Comment by adrienball** [comment]
I'm closing this PR as the tutorial section has been updated in #569 
Thanks @adam-meya  !

**Pull request closed without merging**